components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -973,6 +975,7 @@ private Permissions() {
 
     public static final String API_ANALYTICS = "Analytics.";
     public static final String API_USAGE_ENABLED = API_ANALYTICS + "Enabled";
+    public static final String API_ANALYTICS_POLICY_ENABLED = API_ANALYTICS + "PolicyEnabled";
```

Comment:
This constant does not seem to be used in any other place. May be we can remove.